wordpress-mobile/WordPress-iOS
# Comment: make `rawContent` optional

Ref: #16933, #16876

Self-hosted sites don't have a `rawContent` property, so it is now optional in the Core Data `Comment` model. 

The Comment content for display and for edit now check for either `rawContent` or `content`.

To test:

1. Verify UI tests pass on this PR.

2. On a self-hosted site:
- Log in.
  - Verify the app doesn't crash.
- Go to My Site > Comments.
  - Verify the comments have content.
  - Edit a comment. Verify the content is displayed.

3. On a WordPress site:
- Go to My Site > Comments.
  - Verify the comments have content.
  - Edit a comment. Verify the content is displayed.

## Regression Notes
1. Potential unintended areas of impact
Comment content could be displayed incorrectly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Checked the Comments display as listed above.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
